pkg/plugins/crypto/impl/crypto.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package impl provides the default implementation of pkg/plugins/crypto.Crypto interface
+//
+// It uses github.com/tink/go crypto primitives
+package impl
```

Comment:
This is not a useful package name.

You should restructure - e.g, have a default or impl parent folder and put a crypto inside it.

Reply:
I wanted to separate the interface api from the default implementation... will move up default implementation

Reply:
moved up... and removed impl